src/Components/ReviewCard.jsx

```diff
@@ -1,8 +1,19 @@
+import { useLocation, useNavigate } from "react-router-dom";
 import "./style/ReviewCard.css";
 
-const ReviewCard = ({ title, review_img_url }) => {
+const ReviewCard = ({ review_id, title, review_img_url }) => {
+  const navigate = useNavigate();
+  const location = useLocation();
+  const basePath = location.pathname;
+
+  const handleClick = (event) => {
+    event.preventDefault();
+    if (basePath.includes("reviews")) navigate(`./${review_id}`);
+    else navigate(`reviews/${review_id}`);
+  };
+
   return (
```

Comment:
again here you can just use `Link` which will save you having to import location and navigate and writing out lengthy handleClick's